test/Microsoft.TestPlatform.AcceptanceTests/Extension/RunnnerInfo.cs

```diff
@@ -9,7 +9,13 @@ namespace Microsoft.TestPlatform.AcceptanceTests;
 
 public class RunnerInfo
 {
-    public RunnerInfo(string runnerType, string targetFramework) : this(runnerType, targetFramework, "")
+    // Serialization ctor that enables correct tree view for data source tests
+    private RunnerInfo()
+    {
```

Comment:
`IsNetRunner`, `IsNetFrameworkRunner`, `IsNetTarget`, `IsNetFrameworkTarget` won't be correctly set if this gets deserialized. We probably should switch to Lazy<bool> and calculate those values on first fetch.